backend/inspirehep/mailing/templates/mailing/jobs/mailchimp/job_list.html

```diff
@@ -0,0 +1,39 @@
+{% for job in jobs %}
+    <table class="mcnTextBlock" style="min-width:100%;" width="100%" cellspacing="0" cellpadding="0" border="0">
+        <tbody class="mcnTextBlockOuter">
+            <tr>
+                <td class="mcnTextBlockInner" style="padding-top:9px;" valign="top">
+                    <!--[if mso]>
+                        <table align="left" border="0" cellspacing="0" cellpadding="0" width="100%" style="width:100%;">
+                        <tr>
+                    <![endif]-->
+                    <!--[if mso]>
+                        <td valign="top" width="600" style="width:600px;">
+                    <![endif]-->
+                    <table style="max-width:100%; min-width:100%;" class="mcnTextContentContainer" width="100%"
+                        cellspacing="0" cellpadding="0" border="0" align="left">
+                        <tbody>
+                            <tr>
+                                <td class="mcnTextContent"
+                                    style="padding-top:0; padding-right:18px; padding-bottom:9px; padding-left:18px;"
+                                    valign="top">
+                                    <a title="{{job.position}}"
+                                        href="https://labs.inspirehep.net/jobs/{{job.control_number}}"> {{job.position}}</a>
+                                    ({{ job.institutions[0].value}} - {{', '.join(job.regions)}}) [Deadline:
+                                    {{job.deadline_date}}] {{', '.join(job.ranks)}} - {{', '.join(job.arxiv_categories)}}
```

Comment:
In the example in the US, it displays also when the job was posted, do we not want that anymore?

Reply:
good catch totally missed it, I have updated the PR now